apps/explorer/src/pages/address-result/AddressResult.tsx

```diff
@@ -1,30 +1,40 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { createContext } from 'react';
 import { useParams } from 'react-router-dom';
 
 import { ErrorBoundary } from '../../components/error-boundary/ErrorBoundary';
-import OwnedObjects from '../../components/ownedobjects/OwnedObjects';
 import { TransactionsForAddress } from '../../components/transactions/TransactionsForAddress';
 
+import OwnedCoins from '~/components/OwnedCoins/OwnedCoins';
+import OwnedObjects from '~/components/OwnedObjectsV2/OwnedObjects';
 import { Heading } from '~/ui/Heading';
 import { PageHeader } from '~/ui/PageHeader';
 
+export const AddressContext = createContext('');
+
 function AddressResult() {
     const { id: addressID } = useParams();
 
     return (
         <div className="space-y-12">
             <PageHeader type="Address" title={addressID!} />
-
             <div>
                 <div className="border-b border-gray-45 pb-5 md:mt-12">
                     <Heading color="gray-90" variant="heading4/semibold">
                         Owned Objects
                     </Heading>
                 </div>
                 <ErrorBoundary>
-                    <OwnedObjects id={addressID!} byAddress />
+                    <div
+                        className={`grid w-full grid-cols-1 md:grid-cols-2 
+                        [&>div:first-child]:border-r-[1px] [&>div:first-child]:border-solid 
```

Comment:
I don't love this, I'd rather we just add a div wrapper and style explicitly there, otherwise this component is depending on the DOM layout of the components its rendering which isn't ideal.